source/filtered_table_model.ts

```diff
@@ -19,10 +18,15 @@ export class FilteredTableModel extends TableModel {
    * @param source - The underlying model to filter.
    * @param predicate - The predicate used to filter the table.
    */
-  public constructor(model: TableModel, predicate: Predicate) {
+  public constructor(model: TableModel,
+      predicate?: (row: number, model: TableModel) =>  boolean) {
```

Comment:
The predicate should be ```(model: TableModel, row: number)```. Generally if one parameter depends on another, then it is listed afterwards. In this case the ```row``` depends on the ```model```.